### ISSUE TITLE
Workers unavailable

### ISSUE THREAD

**Issue by dmolina213** [MISP/MISP#4744]
Questions | Answers
-- | --
Type of issue | support
OS version (server) | RedHat
OS version (client) | windows 7
PHP version | 7.2.10
MISP version / git hash | v2.4.107 ()
Browser

I tried a simple process of fetching all events in one of the feeds.

Before performing the action. I checked the status of the workers.
systemctl status misp-workers.service
ΓùÅ misp-workers.service - MISP's background workers
   Loaded: loaded (/etc/systemd/system/misp-workers.service; enabled; vendor pre
set: disabled)
   Active: active (running) since Thu 2019-06-13 20:12:07 UTC; 21min ago
  Process: 18108 ExecStart=/usr/bin/scl enable rh-php72 rh-redis32 rh-mariadb102
 /var/www/MISP/app/Console/worker/start.sh (code=exited, status=0/SUCCESS)
   CGroup: /system.slice/misp-workers.service
           Γö£ΓöÇ18147 bash -c cd '/var/www/MISP/app/Vendor/kamisama/php-resque-
e...
           Γö£ΓöÇ18148 php ./bin/resque
           Γö£ΓöÇ18169 bash -c cd '/var/www/MISP/app/Vendor/kamisama/php-resque-
e...
           Γö£ΓöÇ18170 php ./bin/resque
           Γö£ΓöÇ18192 bash -c cd '/var/www/MISP/app/Vendor/kamisama/php-resque-

I clicked to enable the action. The job was queued in the background and i got this from the jobs queue UI
![image](https://user-images.githubusercontent.com/42444130/59465648-f5fa4d80-8df0-11e9-8470-36b74ab3c74c.png)

I checked the error logs and found this

![image](https://user-images.githubusercontent.com/42444130/59465669-09a5b400-8df1-11e9-8bcd-0da26c27e37a.png)

I need help troubleshooting this issue.

**Comment by rommelfs** [comment]
Your setup is not complete. PHP needs more than the 128 MB memory you allocated to it. The documentation says 512MB, we suggest 1.5GB.
Could you check if that is mentioned in the installation documentation?

**Comment by dmolina213** [comment]
From the untrained eye this is what I see see in  section 0. Should we change those variables? However, it appears this document seems to be changing quite frequently
# php.ini configuration
  upload_max_filesize=50M
  post_max_size=50M
  max_execution_time=300
  memory_limit=512M

**Comment by dmolina213** [comment]
forgive my ignorance but there are a multiple php.ini files on the server. Is there a specific one I need to change ? And I assume restarting apache would be the next action I need to do after changing the variables in php.ini

**Comment by dmolina213** [comment]
I think I got it fixed. Some of the feeds are being fetched and are populated in the events. Feed 1 doesn't seem to work. It maybe some other thing. But I don't notice the error anymore in the error logs when I try to fetch feeds. I edited all the php.ini files in the server and I did # systemctl restart httpd  and # systemctl restart rh-php72-php-fpm.service. Can we leave this issue open for other issues that maybe similar
